modules/layers/src/text-layer/font-atlas-manager.js

```diff
@@ -97,6 +97,22 @@ function setTextStyle(ctx, fontFamily, fontSize, fontWeight) {
   ctx.textAlign = 'left';
 }
 
+/**
+ * Sets the Font Atlas LRU Cache Limit
+ * @param {number} limit LRU Cache limit
+ */
+export function setFontAtlasCacheLimit(limit) {
```

Comment:
Nit: we try not to use verbose error messages because it bloats the bundle size. Maybe consider something like

```js
import {log} from '@deck.gl/core';

log.assert(Number.isFinite(limit) && limit >= CACHE_LIMIT, 'Invalid cache limit');
```

Reply:
Replaced it with your suggestion.